modules/rabbitmq/src/test/java/org/testcontainers/containers/RabbitMqContainerTest.java

```diff
@@ -0,0 +1,32 @@
+package org.testcontainers.containers;
+
+import java.io.IOException;
+import java.net.Socket;
+
+import org.junit.Test;
+
+
+public class RabbitMqContainerTest {
+
+    private RabbitMqContainer container = new RabbitMqContainer();
+
+    @Test
+    public void containerStartsAndPublicPortIsAvailable() {
+        try {
```

Comment:
please move container definition into `try`, it implements AutoCloseable:
```
try (RabbitMqContainer container = new RabbitMqContainer()) {
    container.start();
}
```

Reply:
ok